Mu2eG4/src/Mu2eG4CustomizationPhysicsConstructor.cc

```diff
@@ -44,18 +42,13 @@ namespace mu2e {
     , mu2elimits_(nullptr)
   {}
 
-  Mu2eG4CustomizationPhysicsConstructor::~Mu2eG4CustomizationPhysicsConstructor()
-  {}
-
   void Mu2eG4CustomizationPhysicsConstructor::ConstructParticle()
   {
     // Empty on purpose, for now
   }
 
   void Mu2eG4CustomizationPhysicsConstructor::ConstructProcess()
   {
-    if(wasActivated) { return; }
```

Comment:
Please explain why this is not needed?  Is it because this code is only every called once, in the master thread?

Reply:
That approach was abandoned circa version 10.6+ and the static thread local variable was removed from all the physics constructors.
I still need to check what exactly replaced it.

Reply:
Thanks. That's all I need.